templates/shortcode-receipt.php

```diff
@@ -324,7 +324,7 @@
 					<td><?php echo esc_html( $item->quantity ); ?></td>
 				<?php } ?>
 				<td>
-					<?php echo esc_html( edd_currency_filter( edd_format_amount( $item->total ), $order->currency ) ); ?>
+					<?php echo esc_html( edd_display_amount( $item->total, $order->currency ) ); ?>
```

Comment:
We also need to update lines for adjustments please:
- 127 (fees)
- 102 (discounts)
- 154 (credits)